csv_export.php

```diff
@@ -68,16 +68,8 @@
 
 $t_filename = csv_get_default_filename();
 
-# Send headers to browser to activate mime loading
-
-# Make sure that IE can download the attachments under https.
-header( 'Pragma: public' );
-
-header( 'Content-Type: text/csv; name=' . urlencode( file_clean_name( $t_filename ) ) );
-header( 'Content-Transfer-Encoding: BASE64;' );
-
-# Added Quotes (") around file name.
-header( 'Content-Disposition: attachment; filename="' . urlencode( file_clean_name( $t_filename ) ) . '"' );
+csv_emit_headers( $t_filename );
```

Comment:
Didn't notice that.  I would like to remove this configuration option, add it to obsolete configs and always have Excel/CSV files have this byte order marker.  I can't see why we wouldn't always have it.